postman/collections/graviteeio-am-login-collection-app-version.json

```diff
@@ -5828,6 +5828,56 @@
 									},
 									"response": []
 								},
+								{
```

Comment:
We should be using Jest (formerly Cypress) for our integration tests now.

Reply:
agree, this change is for failing existing scripts